spec/requests/api/v3/subjects/index_spec.rb

```diff
@@ -0,0 +1,847 @@
+require "rails_helper"
+
+describe "GET v3 /subjects" do
+  let(:request_path) { "/api/v3/subjects" }
+  let(:json_response) { JSON.parse(response.body) }
+
+  before do
+    get request_path
+  end
+
+  it "returns the correct data" do
+    expect(json_response).to eq("data" => [
+        {
```

Comment:
Maybe it's worth moving this out to a file - this is a pretty lengthy response and not _overly_ useful to see the whole thing of in the test IMO